src/test/groovy/graphql/DataFetcherTest.groovy

```diff
@@ -98,7 +98,7 @@ class DataFetcherTest extends Specification {
         given:
         def environment = env(GraphQLString)
         when:
-        def result = new PropertyDataFetcher("publicField").get(environment)
+        def result = new FieldDataFetcher("publicField").get(environment)
         then:
         result == "publicValue"
     }
```

Comment:
I don't think the official bean spec is relevant anymore: I think we should support here what is most convenient for the user and least surprising.
I think removing something we supported is not what we wann do.